controllers/transactionsController.js

```diff
@@ -0,0 +1,29 @@
+const transactionsModel = require('../models/transaction')
+const { getUserId } = require('../utils/users')
+
+/**
+ * Collects all transactions and sends only required data for each userId
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const fetchLatest = async (req, res) => {
+  try {
+    const userId = await getUserId(req.params.username)
```

Comment:
Please handle the case where given `username` doesn't exist

Reply:
If user doesn't exist then returned userId will be undefined, on basis of this if condition added